### PR TITLE
Expose ants options

### PR DESCRIPTION
**Pull request by jkralik**

### CODE REVIEW

File: task/queue/queue.go [hunk 1]
```diff
@@ -19,8 +19,9 @@ type TaskQueue struct {
 }
 
 // New creates task queue which is processed by number of workers. Number of task is limited by limit.
-func New(numWorkers, limit int) (*TaskQueue, error) {
-	p, err := ants.NewPool(numWorkers, ants.WithPreAlloc(true), ants.WithNonblocking(true))
+func New(numWorkers, limit int, options ...ants.Option) (*TaskQueue, error) {
+	options = append(options, ants.WithNonblocking(true))
```
Comment (ondrejtomcik) [inline task/queue/queue.go:23]: Shouldn't we define a yaml config struct here in the kit ?

Comment (jkralik) [reply task/queue/queue.go]: Sure. Fixed

File: task/queue/config.go [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+import (
+	"runtime"
+	"time"
+)
+
+// Config configuration for task queue.
+type Config struct {
+	// NumWorkers  number goroutine instances.
+	NumWorkers int `yaml:"numWorkers" json:"numWorkers"`
```
Comment (ondrejtomcik) [inline task/queue/config.go:11]: It's not number of workers, it's goroutinePoolSize . If you add here 10000 it doesn't mean you have 10000 workers handling in parallel goroutines from the pool

Comment (jkralik) [reply task/queue/config.go]: Fixed

File: task/queue/config.go [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+	// Size size of queue. If it exhausted Submit returns error.
+	Size int `yaml:"size" json:"size"`
+	// MaxIdleTime sets up the interval time of cleaning up goroutines, 0 means never cleanup.
+	MaxIdleTime time.Duration `yaml:"maxIdleTime" json:"maxIdleTime"`
+	// PreAlloc indicates whether it should malloc for workers immediately.
+	PreAlloc bool `yaml:"preAlloc" json:"preAlloc"`
```
[the remark this answers is not in the document]
Comment (jkralik) [reply task/queue/config.go]: Fixed

**Pull request merged**